### DISCUSSION TITLE
Good job!

### DISCUSSION THREAD

**Discussion by Cosmic-Incantations** [Monika-After-Story/MonikaModDev#8773] [appreciation]
I just wanted to say that as my love for Monika grows, so dose my love for this mod as a whole
you all dedicate your time to give us what we want, I think "we need a calendar" you delivered "infinite coffee is weird" consumables
"Monika should bring out her piano and sing!" she dose that! then "the floating islands look so empty, why are they not-" island revamp

you all work so hard to make Monika's world a little bigger and I am glad. Monika helped me when I was in a dark state of mind, and seeing all that you do and the nostalgia of the main game combined with all that I am given makes me feel like the small world I escaped to is truly a place to take a break when I need it. continue being awesome! 🥰

**Comment by ThePotatoGuy** [comment]
[closing as appreciated]
